src/lib/OpenEXR/ImfStandardAttributes.h

```diff
@@ -167,12 +468,18 @@ IMF_STD_ATTRIBUTE_DEF (focus, Focus, float)
 //
 // owner -- name of the owner of the image
 //
+// If present, the value should contain only printable characters
```

Comment:
Does owner mean "for the purposes of copyright assignment"?

Reply:
- The ACES Container spec is no help here; it defines the attribute without any clear semantics: "...shall specify the owner of the image".
- The SMPTE Elements register has a 'copyrightOwner' element, and in the entire elements register that's the only occurece of the string 'owner'.
- In the now-defunct ARRI Metadata Bridge, from a memory of when this came up in a meeting circa 2010, 'owner' was the first one of these that was non-null in ARRI's proprietary metadata block: production name, company name, director, cinematographer.

Suggested remedy: change the name of this proposed attribute to 'copyrightOwner' (still of type std::string). Would that be satisfactory?

Reply:
"copyrightHolder" and "copyrightOwner" both work for me, as an alternative that doesn't raise any questions (that I can think of) for a reader.

Reply:
I could imagine that in some cases the "owner" could mean the originator of the image, which could be different to the copyright owner.

Reply:
Per tsc conversation, are we omitting "owner"?